flowmachine/flowmachine/core/server/query_schemas/flowmachine_query.py

```diff
@@ -66,6 +67,7 @@ class FlowmachineQuerySchema(OneOfSchema):
         "unmoving_counts": UnmovingCountsSchema,
         "unmoving_at_reference_location_counts": UnmovingAtReferenceLocationCountsSchema,
         "trips_od_matrix": TripsODMatrixSchema,
+        "per_subscriber_aggregate": PerSubscriberAggregateSchema,
```

Comment:
Any query schema included in `FlowmachineQuerySchema` is exposed as a top-level query (which users can directly run and get results). We don't want that here (subscriber-level queries shouldn't be exposed directly), so PerSubscriberAggregateSchema should only be exposed as an allowed option for appropriate parameters of other queries - in this case, we'll want to add it to joined_spatial_aggregate's `JoinableMetrics`. Thinking about it, it should probably be an allowed input to `HistogramAggregate` as well - so we'll need to re-introduce `HistogrammableMetrics` after all. If there's a way to nest a `OneOfSchema` within another `OneOfSchema` to avoid duplicating the set of schemas in `NumericSubscriberMetricsSchema`, that would be good.